components/analytics-core/org.wso2.carbon.analytics.datasource.core.test/src/main/java/org/wso2/carbon/analytics/datasource/core/AnalyticsDataServiceTest.java

```diff
@@ -58,29 +66,28 @@
 public class AnalyticsDataServiceTest implements GroupEventListener {
 
     private static final int DEFAULT_WAIT_TIME = 60000;
-
+    private static final Log logger = LogFactory.getLog(AnalyticsDataServiceTest.class);
     protected AnalyticsDataService service;
-    
     private boolean becameLeader, leaderUpdated;
-
     private Random randomGenerator;
-               
+
     public void init(AnalyticsDataService service) {
         this.service = service;
         randomGenerator = new Random();
     }
-    
+
     @Test
     public void testMultipleRecordStores() throws AnalyticsException {
-        this.cleanupTable(1, "T1");        
+        logger.info("testMultipleRecordStores");
```

Comment:
Can you explain what information is provided by this log? IMO, this log doen't provide any relevant information. Hence, having it just clutters the code. There are other logs similar to this. Suggesting to remove all.

IMO, unit tests doesn't need logs. Unit test should be strong, fast, & silent. Logs might be (some times must) necessary for integration tests, but they are not for unit tests.

Reply:
I have added this for debugging and monitoring purposes.. as I know we have done this in several other places.

Reply:
As discussed offline, shal we change this log to be more meaningful.
e.g. `logger.debug("Starting Multiple Record Stores test");`